uint16 whichArm
geometry_msgs/Pose hand_Pose